arch/arm/core/aarch32/fatal.c

```diff
@@ -83,9 +84,9 @@ void z_do_kernel_oops(const z_arch_esf_t *esf)
 	unsigned int reason = esf->basic.r0;
 
 #if defined(CONFIG_USERSPACE)
-	if ((__get_CONTROL() & CONTROL_nPRIV_Msk) == CONTROL_nPRIV_Msk) {
```

Comment:
This is refactoring work. @bbolen can you quickly send this in a separate PR that could be reviewed and merged asap?